ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/participant/ParticipantTestContext.scala

```diff
@@ -538,6 +554,20 @@ private[testtool] final class ParticipantTestContext private[participant] (
       ),
     )
 
+  def submitRequest(actAs: List[Party], readAs: List[Party], commands: Command*): SubmitRequest =
```

Comment:
This makes sense. Are we planning on removing the other `submitRequest` that just takes a single party at some point in the future?

Reply:
I think that these two variants do slightly different things because the single-party one uses `party` field, while the multi-party one uses `actAs` field. The `party` field is still present to provide backward compatibility, so it might be beneficial to have some tests covering that too. What do you think?